chart/values.yaml

```diff
@@ -394,10 +394,11 @@ etcd:
   persistence:
     # -- If true, use a Persistent Volume Claim. If false, use emptyDir.
     enabled: true
-    # -- Will define which storageClass to use in etcd's StatefulSets
-    # a `manual` storageClass will provision a hostpath PV on the same node
-    # an empty storageClass will use the default StorageClass on the cluster
-    storageClass: ""
+    # -- Will define which storageClass to use in etcd's StatefulSets. Options:-
+    # <p> - `"openebs-hostpath"` (default) - Will provision openebs local-pv hostpath volume. Needs `Values.openebs-localpv-provisioner.enabled = true`. <br>
+    # - `"manual"` - Will provision a hostpath PV on the same node. <br>
+    # - `""` (empty) - Will use the default StorageClass on the cluster. </p>
+    storageClass: "openebs-hostpath"
```

Comment:
Should this have the release name prefix? 
{{ .Release.Name }}-openebs-hostpath

Reply:
Its the default value in https://github.com/openebs/dynamic-localpv-provisioner/blob/57a57242cdec2dc565a9c94dffed05f71b3bcf7f/deploy/helm/charts/values.yaml#L120

Reply:
Then we can change it as well, just set it.

Reply:
We cannot set the release name from there.